tailwind.config.js

```diff
@@ -7,17 +7,17 @@ module.exports = {
     extend: {
       colors: {
         primary: {
-          200: '#FFDA66',
-          300: '#FFD044',
-          400: '#FFC622',
-          500: '#FFBB00',
-          600: '#EEA600',
-          700: '#DD9100',
-          800: '#CC7E00',
+          200: '#FFDA66', // pale orange, Hue 45, Luminance 70  #fd7 is Hue 45, Luminance 73
+          300: '#FFD044', // orange, Hue 45, Luminance 63  #fc3 is Hue 45, Luminance 60
+          400: '#FFC622', // orange, Hue 45, Luminance 57
+          500: '#FFBB00', // orange, Hue 44, Luminance 50 #fbo is Hue 45, Luminance 50.
+          600: '#EEA600', // orange, Hue 42, Luminance 47
+          700: '#DD9100', // orange, Hue 39, Luminance 43
+          800: '#CC7E00', // darker orange, Hue 37, Luminance 40 #c90 is Hue 45, Luminance 40
         },
         neutral: {
-          150: 'rgb(240, 240, 240)',
-          750: 'rgb(44 44 44)',
+          150: 'rgb(240, 240, 240)', // pale grey, Hue 0, Luminance 94
+          750: 'rgb(44 44 44)', // dark grey, Hue 0, Luminance 71
```

Comment:
This is not correct.

Reply:
good catch on missing commas

Reply:
@haslinghuis what is not correct?  The use of the `//`, or the luminance values?

Reply:
Missing comma's as before.